### ISSUE TITLE
Backup.py assumes path component of S3 URL exists

### ISSUE THREAD

**Issue by pckilgore** [mail-in-a-box/mailinabox#2133]
This line:

https://github.com/mail-in-a-box/mailinabox/blob/99474b348f0dd2632057c18c6a8c4e6464962878/management/backup.py#L211

Causes an exception when attempting to unpack a two-tuple that's actually a single value where the S3 URL does not contain an object path for the backups (or a trailing "/").

Worked around it by adding the "/" to my config.

**Comment by JoshData** [comment]
Yep. This was reported on the forum. I'd appreciate anyone's help making a concise fix to the code.

**Comment by pckilgore** [comment]
I started down this path and ended up realizing, it's not simple!

Surprised no libraries existed so made one: https://github.com/pckilgore/s3_endpoint_parse

What's your feeling on dependencies?  The entire lib is one file, I could also just inline it /shrug.

**Comment by rtuin** [comment]
Just dropped in to say I'm having the same issue and [@pckilgore's suggestion](https://github.com/mail-in-a-box/mailinabox/issues/2133#issue-1273659478) to add a `/` to the bucket name (in the web UI) fixes the `not enough values to unpack (expected 2, got 1)` error for me. 

![Screenshot 2022-07-28 at 10 14 44](https://user-images.githubusercontent.com/360150/181456350-9f813a47-61f6-4f83-b35f-f690b30433b7.png)
